src/pages/EditMonitor.vue

```diff
@@ -664,6 +665,16 @@
                                 </div>
                             </template>
 
+                            <template v-if="monitor.type === 'local-service'">
+                                <div class="my-3">
+                                    <label for="local-service-name" class="form-label">{{ $t("Service Name") }}</label>
+                                    <input id="local-service-name" v-model="monitor.local_service_name" type="text" class="form-control" required placeholder="nginx.service">
+                                    <div class="form-text">
+                                        {{ $t("localServiceDescription") }}
```

Comment:
lets add here (via `i18n-t`) how to debug this monitor.
This way it is much clearer what this monitor does.